JitsiConference.js

```diff
@@ -142,10 +148,11 @@ function JitsiConference(options) {
 
     /**
      * Flag set to <tt>true</tt> when P2P session has been established
-     * (ICE has been connected).
+     * (ICE has been connected) and this conference is currently in the peer to
+     * peer mode (P2P connection is the active one).
      * @type {boolean}
      */
-    this.p2pEstablished = false;
+    this.p2p = false;
```

Comment:
ok, i was thrown off by the explicit mention of the ice state in the comment, viewing it as 2 separate conditions.  maybe then it's sufficient to just say that the flag represents whether or not the p2p session is the one actively being used by this conference?